examples/package.json

```diff
@@ -1,29 +1,39 @@
 {
-  "name": "basic-google-map",
-  "version": "1.0.0",
-  "private": true,
+  "name": "@ubilabs/google-maps-react-hooks-examples",
   "description": "A sample Google Map setup with the MapProvider component.",
   "main": "index.js",
   "scripts": {
-    "clean-examples": "rm -rf public .parcel-cache",
-    "start:map": "PARCEL_AUTOINSTALL=false parcel serve ./basic-google-map/index.html --dist-dir public --port 1234",
-    "start:map-markers": "PARCEL_AUTOINSTALL=false parcel serve ./google-map-with-markers/index.html --dist-dir public --port 1234",
-    "start:geocoding": "PARCEL_AUTOINSTALL=false parcel serve ./geocoding/index.html --dist-dir public --port 1234",
-    "start:places": "PARCEL_AUTOINSTALL=false parcel serve ./places/index.html --dist-dir public --port 1234",
-    "start:autocomplete": "PARCEL_AUTOINSTALL=false parcel serve ./places-autocomplete/index.html --dist-dir public --port 1234",
-    "start:directions": "PARCEL_AUTOINSTALL=false parcel serve ./directions/index.html --dist-dir public --port 1234",
-    "start:distance-matrix": "PARCEL_AUTOINSTALL=false parcel serve ./distance-matrix/index.html --dist-dir public --port 1234"
+    "lint": "eslint './**/*.{ts,tsx}'",
+    "types": "tsc --project tsconfig.json --noEmit",
+    "test": "npm run lint && npm run types",
+    "clean-examples": "rm -rf ./public ../.parcel-cache",
+    "start:map": "npm run clean-examples && cross-env $(cat .env) PARCEL_AUTOINSTALL=false parcel serve ./basic-google-map/index.html --dist-dir public --port 1234 --no-cache",
```

Comment:
This `&& cross-env $(cat .env)` is for the`.env` in the examples folder, right? Maybe we can think of a general `start` script for all examples and put it here. As it is repeating in all lines.

Sth like this:

`"start:example": npm run clean-examples && cross-env $(cat .env)`

or maybe sth. else as `start` as there is some parts missing to really start the example. `pre-start`or sth. like this?